tensorflow/c/c_test_util.cc

```diff
@@ -36,17 +36,31 @@ TF_Tensor* Int8Tensor(const int64_t* dims, int num_dims, const char* values) {
   return t;
 }
 
-TF_Tensor* Int32Tensor(int32_t v) {
-  const int num_bytes = sizeof(int32_t);
-  int32_t* values = new int32_t[1];
-  values[0] = v;
-  return TF_NewTensor(TF_INT32, nullptr, 0, values, num_bytes,
-                      &Int32Deallocator, nullptr);
+TF_Tensor* Int32Tensor(int32_t v, const tensorflow::TensorShape shape) {
+  int num_dims = shape.dims();
+  int64_t* dims = new int64_t[num_dims];
+  for (int i = 0; i < num_dims; ++i)
+    dims[i] = reinterpret_cast<int64_t>(shape.dim_size(i));
+  int64_t num_values = 1;
+  for (int i = 0; i < num_dims; ++i)
```

Comment:
The style guide says braces are optional for single-statement loops, but I prefer braces :) (or put the whole thing on one line without braces)